open-media-match/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+services:
+  app:
+    build: .
+    restart: unless-stopped
+    ports:
+      - 8080:8080
+    environment:
+      OMM_CONFIG: /build/omm_config.py
+    networks:
+      - backend
+    depends_on:
+      - db
+
+  db:
+    image: postgres:latest
+    restart: unless-stopped
+    volumes:
+        - ./.devcontainer/pg-scripts/init-multiple-pg-dbs.sh:/docker-entrypoint-initdb.d/init-multiple-pg-dbs.sh
+        - postgres-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: media_match
```

Comment:
could we use here `"${POSTGRES_USER:-media_match}"` and others

This will allow to use Env variables to replace contents while still allow a default value for these.